Common/ConsoleListener.cpp

```diff
@@ -513,9 +513,6 @@ void ConsoleListener::PixelSpace(int Left, int Top, int Width, int Height, bool
 	bool DAft = true;
 	std::string SLog = "";
 
-	const HWND hWnd = GetConsoleWindow();
```

Comment:
I guess this is fine, but I guess the concern here was in case the user has any way to close the console.  `GetConsoleWindow()` is a Windows API call, whereas the `hWnd` cached value may stop reflecting reality if something externally affects the console window.

-[Unknown]